polkadot/node/subsystem-util/src/controlled_validator_indices.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
```

Comment:
Are we using some tool to insert copy right headers?  Or are we using some tool, e.g. https://github.com/Lucas-C/pre-commit-hooks ?

Reply:
I do it by hand :)